python/tvm/relay/frontend/tensorflow.py

```diff
@@ -44,6 +44,10 @@
 
 __all__ = ["from_tensorflow"]
 
+# By default, TVM convert `tf.matmul` to `nn.dense` op with data tensor non-transposed and weight
+# tensor transposed
+_USE_DENSE_INSTEAD_OF_MATMUL = True
```

Comment:
```suggestion
# The default configurations of Relay TensorFlow frontend.
TF_DEFAULT_CONFIGS = {
  # By default, TVM converts `tf.matmul` to `transpose(weight) + nn.dense`, which introduces
  # unnecessary overhead in weight transpose. Change this flag to False to directly convert to
  #`nn.matmul` to get rid of the overhead. However, please note that `nn.matmul` is in experimental
  # so it may have some performance issues.
  "use_dense": True,
}
```

I reviewed the primary entrypoint and feel we may need to make it more general, as we may have other configurations in the future. Also I used the name "default" to imply that it may be overwritten by user-specified values.